packages/js/src/dashboard/widgets/search-ranking-compare-widget.js

```diff
@@ -0,0 +1,196 @@
+import { useCallback, useMemo } from "@wordpress/element";
+import { __ } from "@wordpress/i18n";
+import { Alert, SkeletonLoader } from "@yoast/ui-library";
+import { useRemoteData } from "../services/use-remote-data";
+import { Widget } from "./widget";
+import { SearchRankingCompareMetric } from "../components/search-ranking-compare-metric";
+import { InfoTooltip } from "../components/info-tooltip";
+import { TooltipContent } from "../components/tooltip-content";
+import { getDifference } from "../transformers/difference";
+/**
+ * @type {import("../index").MetricData} MetricData
+ * @type {import("../index").SearchRankingCompareData} SearchRankingCompareData
+ */
+
+/**
+ * @returns {JSX.Element} The element.
+ */
+const SearchRankingCompareSkeletonLoader = () => {
+	return (
+		<div className="yst-flex yst-flex-col yst-justify-center yst-items-center  @6xl:yst-flex-row @6xl:yst-justify-evenly rtl:yst-flex-row-reverse ">
```

Comment:
These are shared with the actual, right? Maybe a constant to make it impossible to update the other if changing it?